lib/pf/api.pm

```diff
@@ -459,6 +461,127 @@ sub validate_argv {
     return 1;
 }
 
+
+=head2 add_person
+
+Add a new person
+
+=cut
+
+sub add_person : Public {
+    my ($class, %params) = @_;
+    my $logger = pf::log::get_logger();
```

Comment:
I would like to have something like that:
    my @require = qw(pid);
    my @found = grep {exists $params{$_}} @require;
    return unless validate_argv(\@require,  \@found);